RecoTracker/TkDetLayers/src/Phase2OTECRingedLayer.cc

```diff
@@ -128,7 +128,7 @@ Phase2OTECRingedLayer::groupedCompatibleDetsV( const TrajectoryStateOnSurface& s
   }
 
   //order is odd rings in front of even rings
-  constexpr int ringOrder[NOTECRINGS]{0,1,0,1,0,1,0,1,0,1,0,1,0,1,0};
```

Comment:
can we keep "const" here?
This is logically what it was before the change proposed here.